ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -22,6 +23,10 @@ import (
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
 )
 
+var (
+	vpcUnavailable = regexp.MustCompile("503|service unavailable|server is down for maintenance or overloaded")
```

Comment:
I have seen 503 for many other APIs (especially instance listing).
Why only this particular? If in future we are planning to handle this for all other resources then I think we should do it in power-go-client instead of here.
cc @rmoralesjr